### ISSUE TITLE
Assembler macros generate  "Operation code not found - SAVE" problem messages in editor

### ISSUE THREAD

**Issue by ecbyahoo** [eclipse/che-che4z-lsp-for-hlasm#274]
**Describe the bug**
Unable to successfully configure libs for hlasm plugin.

**To Reproduce**
I have read the configuration of libraries on the wiki several times but no luck in getting the hlasm plugin to find my macro files.
https://github.com/BroadcomMFD-DevOps/che-che4z-lsp-for-hlasm/wiki/Configuration-of-libraries
But so far, no luck.  I'm still not getting my proc_grps.json file correct.  Is this an error in my json file?

**Expected behavior**
SAVE.mac file is never found by the hlasm plugin no matter what settings I put in proc_grps.json

**Screenshots**

![image](https://user-images.githubusercontent.com/107581059/190875029-bc9a794b-d187-4ddf-ad86-8f48d843e1b9.png)

**Platform**

Visual Studio Code environment:

Version: 1.71.2 (user setup)
Commit: 74b1f979648cc44d385a2286793c226e611f59e7
Date: 2022-09-14T21:03:37.738Z
Electron: 19.0.12
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19044
Sandboxed: No

Hlasm plugin:  V1.4.0

**Additional context**

pgm_conf.json is:
```
{
  "pgms": [
    {
      "program": "HELLO",
      "pgroup": "MF"
    }
  ]
}
```
proc_grps.json is:
```
{
  "pgroups": [
    {
      "name": "MF",
      "libs": ["D:\\VSCODEWS\\HLASM",
        {
          "path": "D:\\VSCODEWS\\HLASM",
          "macro_extensions": [
            ".mac"
          ]
        }
      ]
    }
  ],
  "macro_extensions": [
    ".hlasm"
  ]
}
```
Source code directory is:

 Volume in drive D is DATAPART1
 Volume Serial Number is 428A-39E8

 Directory of D:\vscodews\hlasm

09/17/2022  01:07 PM    <DIR>          .
09/17/2022  01:07 PM    <DIR>          ..
09/17/2022  12:52 PM    <DIR>          .hlasmplugin
09/17/2022  12:58 PM    <DIR>          .vscode
09/17/2022  01:00 PM                60 HELLO.hlasm
09/17/2022  12:55 PM                68 SAVE.mac
               2 File(s)            128 bytes
               4 Dir(s)  462,793,871,360 bytes free

**Comment by slavek-kucera** [comment]
Hello @ecbyahoo,

I think the problem is in the pgm_conf.json configuration file. You specified "HELLO", while the file in fact has an extension.
So, you can either just append the extension to the program name in the configuration file ("HELLO.hlasm"), or perhaps you may want to try pattern based configuration - "*" will match all files in the workspace folder.

**Comment by ecbyahoo** [comment]
That was the problem!  I'm up and running!  I think I see why.  This was fresh install and I'm a first time user so when I edited a file called HELLO the editor was in plain text mode.  I didn't know about clicking on the program type in the bottom right corner to change the type to HLASM.  I was prompted to create a pgm_conf.json just for "HELLO" and I clicked yes.  This was before I had saved it to disk and before I switched the language type to HLASM.  So that original pgm_conf.json file has been sitting there with no extension on it from the beginning.  Thank you very much sir!
